packages/web/src/components/modal-radio/ModalRadioGroup.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react'
```

Comment:
[packages/stems/src/components/RadioButtonGroup/RadioButtonGroup.tsx](https://github.com/AudiusProject/audius-client/blob/937b1886cdb932649571cb13db22640b47d07e12/packages/stems/src/components/RadioButtonGroup/RadioButtonGroup.tsx)?

Reply:
this is different.
the radio button here is new in web. no bandwidth to move to stems atm but that's the goal, and also have a better name for it, but for now it's only in some modals.

Reply:
Are they different only by style? The `RadioButtonGroup` component is hopefully versatile enough if not the `RadioButton` itself